README.md

```diff
@@ -27,20 +27,29 @@ Dependencies in Debian trixie:
 build-essential meson ninja-build pkg-config qt5-qmake qttools5-dev-tools qtbase5-dev libtermpaint-dev libtuiwidgets-dev libposixsignalmanager-dev
 ```
 
+Installing dependencies in Fedora:
+```
+sudo dnf install gcc gcc-c++ meson ninja-build pkg-config qt5-qtbase-devel qt5-linguist
```

Comment:
just like Debian, please only list the packages not how it can be installed. 

In my test ci (Fedora33 - 38) I install the following: 
```
dnf -y groupinstall "Development Tools" && \
dnf -y install git devscripts ninja-build pkgconfig qt5-qtbase-devel which python-setuptools python3-testresources chrpath
```
I assume that `qt5-linguist` is not needed?